build.sh

```diff
@@ -46,10 +46,10 @@ SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # When updating the images, consider updating pull-all-couchdbdev-docker
 # script as well
 #
-DEBIANS="debian-buster debian-bullseye"
+DEBIANS="debian-buster debian-bullseye debian-bookworm"
 UBUNTUS="ubuntu-bionic ubuntu-focal ubuntu-jammy"
 CENTOSES="centos-7 almalinux-8 almalinux-9"
-XPLAT_BASE="debian-bullseye"
+XPLAT_BASE="debian-bookworm"
```

Comment:
Let's not switch default XPLAT_BASE yet, let's keep it as bullseye for now.

Reply:
ok, changed it back.